redbot/cogs/mod/mutes.py

```diff
@@ -207,6 +207,14 @@ async def voice_mute(self, ctx: commands.Context, user: discord.Member, *, reaso
             await ctx.send(
                 _("Muted {user} in channel {channel.name}").format(user=user, channel=channel)
             )
+            try:
+                await user.move_to(channel)
+            except discord.Forbidden:
+                await ctx.send(
+                    _(
+                        "Because I don't have the move members permission, this will take into effect when the user rejoins."
```

Comment:
Permission name should be title-cased same as it is presented in Discord's UI.
```suggestion
                        "Because I don't have the Move Members permission, this will take into effect when the user rejoins."
```